pkg/fileservice/cache_replacement/types.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache_replacement
```

Comment:
Conventionally we don't put underscores in package names, cachereplacement or simply cachepolicy may be better.
Package names `lru` and `clock` are too commonly used as variable names, I suggest `lrupolicy` and `clockpolicy` to avoid aliasing when importing.

Reply:
Sure. I have updated the package names accordingly.